src/plot_api/plot_api.js

```diff
@@ -1624,6 +1624,11 @@ function _restyle(gd, aobj, _traces) {
                     flags.docalc = true;
                 }
 
+                // some attributes declare an 'editType' flaglist
+                if(valObject.editType === 'docalc') {
+                    flags.docalc = true;
+                }
```

Comment:
👍 
I guess later this can change to something like 
```js
if(valObject.editType) {
    valObject.editType.split('+').forEach(function(flag) {
        flags[flag] = true;
    });
}
```
But don't need to do that now - lets wait until we have validation of `editType`, perhaps in plotschema_test?